.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-<!--- Describe your changes in detail -->
+I am Biswashree Dey and I request to join as a maintainer.
```

Comment:
Sorry, this is the wrong file where the change has been made. This should have been done on the template that comes up on raising a pull request. Also, you would like to append your github username in the CODEOWNERS file.